cli/README.md

```diff
@@ -54,6 +54,12 @@ After creating the new key, you can specify its value using the `--accessKey` fl
 code-push login --accessKey <accessKey>
 ```
 
+After using the key, if you want to log out of your current session, but still want to be able to reuse the same key for future logins, run the following command:
```

Comment:
How about:

If you want to log out of your current session without invalidating the key for future sessions, run the following command: